openhands/server/session/conversation_init_data.py

```diff
@@ -9,4 +9,4 @@ class ConversationInitData(Settings):
     """
 
     github_token: str | None = Field(default=None)
-    selected_repository: str | None = Field(default=None)
+    selected_repository: SecretStr | None = Field(default=None)
```

Comment:
```suggestion
    selected_repository: str | None = Field(default=None)
```
We probably want the `github_token` above and not so much this one

Reply:
Ah I went too fast and mistyped on these 😅